wp-lazy-loading.php

```diff
@@ -32,13 +32,15 @@
 function _wp_lazy_loading_initialize_filters() {
 	// The following filters would be merged into core.
 	foreach ( array( 'the_content', 'the_excerpt', 'comment_text', 'widget_text_content' ) as $filter ) {
-		// After parsing blocks and shortcodes.
-		add_filter( $filter, 'wp_add_lazy_load_attributes', 25 );
+		add_filter( $filter, 'wp_filter_content_attachment_images' );
```

Comment:
Still thinking this should run after shortcodes are parsed. Why would images added with shortcodes be excluded? Is there any specific reason?

Reply:
The filter is on 10 solely because the existing filter is on 10. I could ask as well why the existing filter for srcset and sizes doesn't cover shortcodes.

I think it's perfectly fine to move this to a later priority, but arguably the benefits of covering shortcodes (especially with blocks being the primary method for dynamic content) would be rather low.

What do we decide here between changing an existing priority (srcset and sizes would suddenly apply to shortcodes) or not (we wouldn't be covering shortcodes)?

Reply:
Yeah, if we wanted to catch shortcodes content, we cannot reuse the same regex. You're right about shortcodes being "on the way out", so probably can ignore this.